<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emailLogin_subPage</class>
 <widget class="QWidget" name="emailLogin_subPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lt_emailaddress">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lt_verifycode">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="btn_getcode">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>获取验证码</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>邮箱验证码登录</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_emaillogin">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
